sympy/utilities/tests/test_lambdify.py

```diff
@@ -534,8 +534,8 @@ def test_python_div_zero_issue_11306():
     p = Piecewise((1 / x, y < -1), (x, y < 1), (1 / x, True))
     f = lambdify([x, y], p, modules='numpy')
     numpy.seterr(divide='ignore')
-    assert float(f(numpy.array([0]),numpy.array([0.5]))) == 0
-    assert str(float(f(numpy.array([0]),numpy.array([1])))) == 'inf'
+    assert float(f(numpy.array([0]).squeeze(), numpy.array([0.5]).squeeze())) == 0
+    assert str(float(f(numpy.array([0]).squeeze(), numpy.array([1]).squeeze()))) == 'inf'
```

Comment:
Is it not possible to just make a numpy scalar:
```
In [1]: float(numpy.array([0]))
<ipython-input-1-3934c0a777a5>:1: DeprecationWarning: Conversion of an array with ndim > 0 to a scalar is deprecated, and will error in future. Ensure you extract a single element from your array before performing this operation. (Deprecated NumPy 1.25.)
  float(numpy.array([0]))
Out[1]: 0.0

In [2]: float(numpy.array(0))
Out[2]: 0.0
```
Alternatively why is this test using `str(float(...))` at all?

Reply:
I changed it to `numpy.array(0)`, and removed the usage of `str(float(...))`.